vllm/model_executor/layers/vocab_parallel_embedding.py

```diff
@@ -193,12 +235,17 @@ def __init__(self,
                                                self.tp_size)
         self.embedding_dim = embedding_dim
 
-        linear_method = None
+        embedding_method, linear_method = None, None
         if quant_config is not None:
-            linear_method = quant_config.get_quant_method(self)
+            methods = quant_config.get_quant_method(self)
+            if methods is not None:
+                embedding_method, linear_method = methods
+        if embedding_method is None:
+            embedding_method = UnquantizedEmbeddingMethod()
```

Comment:
Is there a better way to add two quant methods for GGUF?
(Maybe separate `get_quant_method` to `get_linear_method` and `get_embed_method`?)